static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -6,11 +6,24 @@ import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import once from 'lodash/once';
 import { prebid } from 'commercial/modules/prebid/prebid';
 
+const isGoogleWebPreview: () => boolean = () => {
+    try {
+        return navigator.userAgent.indexOf('Google Web Preview') > -1;
```

Comment:
Might be better to check for `navigator`, `userAgent`, etc rather than relying on the catch block